### DISCUSSION TITLE
HTTP links always use HTTPS

### DISCUSSION THREAD

**Discussion by dudanov** [community/community#55183] [Mobile]
HTTP links always used HTTPS protocol.

**Comment by aconsuegra** [comment]
Hi @dudanov, could you ellaborate a bit more so I could understand what the problem you are facing is? Thanks!

**Reply by dudanov** [reply]
Hi!

When following a link in a mobile application to an HTTP page (not HTTPS), the browser reports an error establishing an HTTPS connection (ERR_SSL_PROTOCOL_ERROR in Google Chrome). Address bar explicitly indicates the use of HTTPS.

For example, here is [my repository](https://github.com/dudanov/docker-openscada). There are HTTP links on the main page in the table, you can test it.

Thanks.

**Reply by aconsuegra** [reply]
Thanks! If I understand the problem, this seems to be an issue with the browser on your phone, but not with the native GitHub Mobile apps?

**Reply by dudanov** [reply]
No. Problem with any browser in Android OS native GitHub App. Now I tried to change the default browser. Problem still exists.
